docs/sections/user_guide/cli/mode_config.rst

```diff
@@ -434,16 +434,16 @@ and an additional supplemental YAML file ``values2.yaml`` with the following con
 .. note:: In recognition of the different sets of value types representable in each config format, ``uw`` supports two format-combination schemes:
 
    1. **Output matches input:** The format of the output config matches that of the input config.
-   2. **Input is YAML:** If the input config is YAML, any output format may be requested. In the worst case, values always have a string representation, but note that, for example, the string representation of a YAML sequence (Python ``list``) in an INI output config may not be useful.
+   2. **Output or input is YAML:** If either config is YAML, any other format may be requested. In the worst case, output values always have a string representation, but note that, for example, the string representation of a YAML sequence (Python ``list``) in an INI output config may not be useful.
 
    In all cases, any supplemental configs must be in the same format as the input config and must have recognized extensions.
 
    ``uw`` considers invalid combination requests errors:
 
    .. code-block:: text
 
-      $ uw config realize --input-file b.nml --output-file a.yaml
-      Output format yaml must match input format nml
+      $ uw config realize --input-file b.nml --output-file a.ini
+      Output format ini must match input format nml unless either is YAML
```

Comment:
That's a weird message: Obviously neither is `YAML` if one is `ini` and the other is `nml`. There's bound to be a better way to state this.

Reply:
The user guide messages need to be reinstated, they were lost in resolving conflicts, but I definitely was not happy with this phrasing, so I'm working on it

Reply:
Possibly better phrasing has been updated